AfricanReligions/localisation/afr_event_texts.csv

```diff
@@ -78,4 +78,19 @@ EVTDESC_MNM_3227_AFRICAN;As I step up to the [Root.GetReligiousCeremonyLocation]
 EVTDESC_MNM_3229_BORI;I walk to the center of the ceremony and make a quick decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin singing [Root.Religion.GetScriptureName]. I roll my eyes back as I sing, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
-EVTDESC_MNM_3229_AFRICAN;I step forwards and make a bold decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin quoting fragments of [Root.Religion.GetScriptureName]. I roll my eyes back as I speak, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
\ No newline at end of file
+EVTDESC_MNM_3229_AFRICAN;I step forwards and make a bold decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin quoting fragments of [Root.Religion.GetScriptureName]. I roll my eyes back as I speak, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
+GROUPSTITLE;Western Traditions;;;;;;;;;;;;;x
+EVTDESCGROUPS1000;Many are the religious traditions of the Western Sahel, men of different lands have different names for the primordial twins, the star Sirius and even for God himself. Now I have the option to choose between one of three traditions. That of the Serer, with its complex rituals and spiritualism, that of the Dogon, masters of astronomy, or that of the Malinke, conquerors and masters of the forge.;;;;;;;;;;;;;x
+EVTOPTCNLGROUPS1000;I wish to choose a different path;;;;;;;;;;;;;x
+EVTOPTAGROUPS.1000;The Serer tradition;;;;;;;;;;;;;x
+EVTOPTBGROUPS.1000;The Dogon tradition;;;;;;;;;;;;;x
+EVTOPTCGROUPS.1000;The Malinke tradition;;;;;;;;;;;;;x
+GROUPS1001TITLE;The Serer;;;;;;;;;;;;;x
+EVTDESCGROUPS1001;The Serer practice �The way of the Divine�, the most complex religious tradition in all of West Africa. They refer to the supreme god as Roog, the primordial twins as Yaab and Yop, and the great star Sirius as Yoonir. Their mastery of spiritual knowledge is greater than that of any other and depends on the interaction between the three worlds: the invisible world, the terrestrial world and the nocturnal world.;;;;;;;;;;;;;x
+EVTOPTAGROUPS1001;This truly is the way of the Divine;;;;;;;;;;;;;x
```

Comment:
The localizations that we were given often don't have ending punctuation (periods, etc.) but most CK2 event options do.  So I generally add a period if it's missing.